### PR TITLE
add ZS 9 repo and PHP 7 param value

### PR DESCRIPTION
**Pull request by fgerards**

### CODE REVIEW

**Comment by davidl-zend** [comment]
Thanks @fgerards - looks good

File: manifests/repo/redhat.pp [hunk 1]
```diff
@@ -7,18 +7,18 @@
   }
 
   yumrepo { 'Zend':
-    baseurl  => "http://repos.zend.com/zend-server/${zendserver::zend_server_version}/rpm/${::architecture}",
```
Comment (ubellavance) [inline manifests/repo/redhat.pp]: That breaks backwards compatibility with older ZS versions. Version 7, for example: http://repos.zend.com/zend-server/7.0.0_update8/ is http://repos.zend.com/zend-server/7.0.0_update8/rpm/

Comment (mkherlakian) [inline manifests/repo/redhat.pp]: @ubellavance can anything be passed as the zendserver::zend_server_version, or is it only a version number such as x.y?
The convention on the Zend repo is for the x.y version to symlink to whatever latest version's available in the repo, in that case 7.0 -> 7.0.0_update8. However I can see that the flexibility to pin to a specific version would be helpful.

Comment (ubellavance) [inline manifests/repo/redhat.pp]: I think that it is a single digit only, but 7.0 is also not in rpm_apache2.4 so I think that someone could not install 7.

**Pull request merged**